markdownlint.js

```diff
@@ -321,11 +328,20 @@ function lintAndPrint(stdin, files) {
   printResult(lintResult);
 }
 
-if ((files.length > 0) && !options.stdin) {
-  lintAndPrint(null, diff);
-} else if ((files.length === 0) && options.stdin && !options.fix) {
-  const getStdin = require('get-stdin');
-  getStdin().then(lintAndPrint);
-} else {
-  program.help();
+function main() {
```

Comment:
What do you think about just putting this inside the try?

Reply:
done!